dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -875,6 +876,38 @@ portalRouter.route('/cron-jobs').delete(cronJobsController.deleteAllEstoreLogs);
 portalRouter.route('/gef/mandatory-criteria/latest').get(mandatoryCriteria.getLatestGefMandatoryCriteria);
 portalRouter.route('/gef/mandatory-criteria/version/:version').get(mandatoryCriteria.getGefMandatoryCriteriaByVersion);
 
+/**
+ * @openapi
+ * /utilisation-reports:
+ *   put:
+ *     summary: Save utilisation report data
+ *     tags: [Portal - Utilisation Report Service]
+ *     description: Save utilisation report data
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *     responses:
+ *       201:
+ *         description: OK
+ *         content:
+ *           application/json:
+ *             schema:
+ *               allOf:
+ *                 - type: object
+ *                   properties:
+ *                     dateUploaded:
+ *                       example: 2023-10-27T08:07:40.028Z
+ *       500:
+ *         description: Internal server error
+ *       400:
+ *         description: Invalid payload
+ *       429:
+ *         description: Server conflict
```

Comment:
Reminder to update this to 409 if we update the controller (as per your comment).